### ISSUE TITLE
Incompatibility with RainLab.User 3.0

### ISSUE THREAD

**Issue by cyril-design** [OFFLINE-GmbH/oc-mall-plugin#1147]
Hi.
Since I updated RainLab.User 3.0 and Mall 3.3 I've got an error:
`Authentication user provider [user] is not defined.`
`~/vendor/laravel/framework/src/Illuminate/Auth/CreatesUserProviders.php line 39`

I've tried with a simple layout with the cart component.
Idem with Products page in backend.

And in the Users page in backend:

``SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.offline_mall_customer_group_id' in 'where clause' (SQL: select `users`.*, (select name from `offline_mall_customer_groups` where `users`.`offline_mall_customer_group_id` = `offline_mall_customer_groups`.`id`) as `customer_group`, (select name from `user_groups` where `users`.`primary_group_id` = `user_groups`.`id`) as `primary_group` from `users` limit 20 offset 0)``

Thanks !

PHP: 8.2
October CMS: v3.6.22
Rainlab User: v3.0.5
Offline Mall: v3.3.1
Laravel Framework: 9.52.16

**Comment by tobias-kuendig** [comment]
Could yo uplease post the full stack trace? Thank you!

**Comment by SamBrishes** [comment]
Hello,

while upgrading `RainLab.User`, did you possibly uninstalled the v2 version of the plugin beforehand or rollback' the migrations?

Did you already had a previous version of the Mall plugin installed, or was it a completely fresh installation?

Sincerely,\
Sam.

**Comment by cyril-design** [comment]
Thanks for your time Sam.
1. I have just done a October:update (before Mall v3.1: Rainlab.user cannot be upgrade to major version)
2. Our app runs Mall plugin since about 2 years

Production app runs ok with Mall v3.1 and Rainlab.User v2.
I have mirrored prod app on an another local machine and I have got the same problems when I update Mall + User

Have you encountered any other similar problems?

Thanks !
